package.json

```diff
@@ -46,11 +48,13 @@
     "gruntfile-editor": "^0.2.0",
     "inquirer": "^0.8.0",
     "isbinaryfile": "^2.0.0",
+    "istextorbinary": "^1.0.2",
     "lodash": "^2.4.1",
     "mem-fs-editor": "^1.0.0",
     "mime": "^1.2.9",
     "mkdirp": "^0.5.0",
     "nopt": "^3.0.0",
+    "prettysize": "0.0.3",
```

Comment:
Use [`pretty-bytes`](https://github.com/sindresorhus/pretty-bytes). It's more correct.